tests/snapshots/test_sampling_with_rate_limit_3.json

```diff
@@ -9,15 +9,14 @@
     "type": "",
     "error": 0,
     "meta": {
-      "_dd.p.dm": "-3",
+      "_dd.p.dm": "-0",
```

Comment:
Here rate limiting is NOT applied since trace sample rules AND a global sample rate is NOT set, the span underdoes default sampling and is set to auto keep. The agent will determine the real rate limits.


This is the regression test for this PR!